PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/ValidationUtils.java

```diff
@@ -106,4 +128,15 @@ public static void validateAppServicePlanName(String appServicePlan) {
             throw new IllegalArgumentException(String.format("App Service Plan Name should match %s", APP_SERVICE_PLAN_NAME_PATTERN));
         }
     }
+
+    private static void cacheAndThrow(Map exceptionCache, Object key, IllegalArgumentException exception) {
```

Comment:
Modify last param to be a string, and in last statement: throw IllegalArgumentException (exception);

Reply:
Yes, has fixed it in latest commit